src/gluonts/mx/distribution/lds.py

```diff
@@ -198,6 +199,7 @@ def log_prob(
         self,
         x: Tensor,
         scale: Optional[Tensor] = None,
+        output_transform: Optional[ComposedBijectionHybridBlock] = None,
```

Comment:
Same as for the `DeepStateEstimator`: couldn't one use whatever `Bijection` here in principle?